exercicios_projeto/Account/Account.js

```diff
@@ -0,0 +1,58 @@
+//const { Client } = require("../Client/Client")
+
+class Account {
+    client
+    numberAccount
+    balance
+    //pixKey
+
+    constructor(client, numberAccount){
+        //if(!(client instanceof Client)){
+        //    console.log(`${this.client}, informe um cliente válido!`)
+        //    return
+        //}
+
+        this.client = client
+        this.numberAccount = numberAccount
+        this.balance = 0
+        //this.pixKey = undefined
+    }
+
+    deposit(amount){
+        this.balance = this.balance + amount
```

Comment:
Oie! Nessa linha tu pode fazer this.balance += amount! Aí tu não precisa repetir o this.balance